plugins/thread.py

```diff
@@ -32,13 +32,14 @@ def run(self):
         project_directory = os.path.dirname(self.process_manager.board.GetFileName())
 
         try:
+            webbrowser.open("file://%s" % (temp_dir))
             # generate gerber
             self.progress(5)
-            self.process_manager.generate_gerber(temp_dir)
+            self.process_manager.generate_gerber(temp_dir + "_g")
```

Comment:
I believe that you verify that this folder exists.
could you please add `os.makedirs(temp_dir + "_g")` after line 29?